### DISCUSSION TITLE
what is your PC spec

### DISCUSSION THREAD

**Discussion by arthurwu4work** [coqui-ai/TTS#2608] [General]
Hi @erogol , 

Can you tell me what are the PC specifications you used to train the models ?
(CPU、GPU、RAM...etc)
and how long it takes to train a multi-speaker tts model ?

thank you

**Comment by erogol** [comment]
I use a regular laptop and rely on the cloud for training. For personal projects colab or paperspace or AWS. Depending on the data and model required time varies a lot. But for instance, training one VCTK would mean a week of single GPU time.

**Comment by arthurwu4work** [comment]
Hi @erogol 

Thank you for your prompt reply!

"a week of single GPU time"
What kind of GPU are you using ?

thank you
